kubetest/build.go

```diff
@@ -42,7 +43,11 @@ func (b *buildStrategy) String() string {
 // Set to --build=B or buildDefault if just --build
 func (b *buildStrategy) Set(value string) error {
 	if value == "true" { // just --build, choose default
-		value = buildDefault
+		if runtime.GOARCH == "amd64" {
```

Comment:
Just out of curiosity. Assuming that we want to only change value for "arm", why are we changing value for everything different than "amd64"?

Reply:
I think changing this for non-amd64 is the correct thing to do for now, the build image cross compiles to other arches but only from amd64 IIRC

Reply:
Thank you @BenTheElder .
Very clear.